packages/dev/s2-docs/src/CodePlatter.tsx

```diff
@@ -313,6 +319,31 @@ function ShadcnDialog({registryUrl}) {
   );
 }
 
+function CodeSandboxDialog({getExampleFiles, codeRef, files, urls, entry, deps, type}) {
+  return (
+    <Dialog size="L">
+      {({close}) => (<>
+        <Heading slot="title">Create a CodeSandbox</Heading>
+        <Content>
+          <p>This will open a new tab with a CodeSandbox containing this example. If you see a 403 error, please log in to CodeSandbox and try again. If you are already logged in, please logout and log back in and then try again.</p>
```

Comment:
Maybe mention `Anonymous users are not allowed to create devboxes.` instead of `403` here.